src/elements/card/src/index.tsx

```diff
@@ -33,12 +33,16 @@ const Card: React.FC<Props> = ({
         borderColor: 'grey-20',
         padding: theme.card.flexDirection ? '0' : 'xs',
         marginTop: '0',
-        marginRight: 'xs',
+        marginRight: ['0', 'xs'],
```

Comment:
unrelated to this PR, but does this cause a bug on desktop where there's padding on the very right element?